sdp.go

```diff
@@ -389,11 +389,18 @@ func addTransceiverSDP(d *sdp.SessionDescription, isPlanB, shouldAddCandidates b
 
 	parameters := mediaEngine.getRTPParametersByKind(t.kind, directions)
 	for _, rtpExtension := range parameters.HeaderExtensions {
-		extURL, err := url.Parse(rtpExtension.URI)
-		if err != nil {
-			return false, err
+		for _, te := range t.getHeaderExtensions() {
```

Comment:
This is for point 2. Cross check with transceiver's extmap and ad only the intersection in the answer.